spec/bolt/logger_spec.rb

```diff
@@ -42,17 +42,16 @@ def initialize(*args)
   end
 
   describe '::configure' do
-    let(:config) {
-      Bolt::Config.new(
-        log: {
-          'file:/bolt.log' => {
-          },
-          'file:/debug.log' => {
-            level: :debug,
-            append: false
-          }
+    let(:appenders) {
+      {
+        'console' => {},
```

Comment:
Why does this need to exist now?

Reply:
In the CLI class, the argument passed to `Logging.configure` comes from the Config object, which always has a default for `console`. The tests were previously configuring a Config object with some files and then passing it in. Now we're bypassing the Config object, so this was changed to explicitly include the `console` section.

But it seems like if I just remove it and update the one test that is _counting_ the number of appenders, it works. I'll do that.